### DISCUSSION TITLE
Issue with RL-820A main stream

### DISCUSSION THREAD

**Discussion by hagak** [blakeblackshear/frigate#1205] [Ask A Question]
I recently added an RLc-820A to my stable of cameras.  Running into an issue getting Frigate to handle the main h265 stream.

I get the below error.  Using the sub stream which is H264 I have no issues.

 ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures

**Comment by bmutari** [comment]
I was having the same problem, here's how I fixed it: https://github.com/blakeblackshear/frigate/discussions/1082

**Reply by hagak** [reply]
Yeah I found your thread and tried those input_args but no change.  Where you getting that same memoryview error?
